### PR TITLE
Add getUserBadgeTimestamp

### PR DESCRIPTION
**Pull request by policetonyR**
This feature uses the https://badges.roblox.com/docs#!/BadgeAwards/get_v1_users_userId_badges_awarded_dates api

It allows you to see the timestamp of when a user is awarded a badge.

### CODE REVIEW

**Comment by Neztore** [comment]
What do you think about renaming this method to getBadgeAwardedTimestmap or similar, like getBadgeAwarded or even getBadgeAwarded date?

Just seems more clear, in my opinion. Open to whatever, though!

**Comment by policetonyR** [comment]
I only named it this because I did not know what else to name it. You could maybe do getBadgeAwardedTimestamps since it takes an array like getPresences.

File: lib/badges/getUserBadgeTimestamp.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Includes
+const http = require('../util/http').func
+
+// Args
+exports.required = ['userId','badgeId']
+
+// Define
+const getBadgeTimestamp = (userId, badgeId) => {
```
Comment (Neztore) [inline lib/badges/getUserBadgeTimestamp.js:8]: Rename this method (The file and the function itself plus the typings) to getAwardedTimestamps.

I do not think there are any other assets that can be "awarded" and this avoids plurality surrounding "badge(s)" in the method name.

**Review by Neztore — CHANGES_REQUESTED** [review]

File: lib/badges/getUserBadgeTimestamp.js (hunk 1 above)
Comment (policetonyR) [reply lib/badges/getUserBadgeTimestamp.js]: Added

**Review by Neztore — APPROVED** [review]

**Pull request merged**